notification/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commercetools-adyen-integration-notification",
-  "version": "9.10.0",
+  "version": "9.12.0",
```

Comment:
What about package-lock in notification?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1011/commits/e65a22d5c9b7eb29504ccc6a98587a1be23e3215